app/graph/types/project_media_type.rb

```diff
@@ -394,4 +394,16 @@ def articles_count
     count += 1 if object.fact_check
     count
   end
+
+  field :relevant_articles, ::ArticleUnion.connection_type, null: true
+
+  def relevant_articles
+    object.get_similar_articles
+  end
+
+  field :relevant_articles_count, GraphQL::Types::Int, null: true
+
+  def relevant_articles_count
+    object.get_similar_articles.count
+  end
```

Comment:
I assume this comes from the merge with @melsawy 's branch.

Reply:
yes @caiosba